src/test/java/org/apache/sling/pipes/internal/MovePipeTest.java

```diff
@@ -69,6 +69,19 @@ public void testMoveNodeWithOverwrite() throws Exception {
         Assert.assertFalse("source node path should have gone", session.nodeExists(PATH_FRUITS + APPLE_NODE_PATH));
     }
 
+    @Ignore //move operation is not supported yet by MockSession
```

Comment:
cannot we try with MOCK_JCR or JCR_OAK (cc @stefanseifert )?

Reply:
thx Nicolas for the suggestion , I tried using JCR_MOCK which does support moving the nodes , I ll raise another commit with the test